cmd/otelcol/config/collector/agent_config.yaml

```diff
@@ -114,15 +114,14 @@ processors:
     detectors: [gcp, ecs, ec2, azure, system]
     override: true
 
-  # Optional: The following processor can be used to add a default "deployment.environment" attribute to the logs and 
+  # The following processor is used to add a default "deployment.environment" attribute to the logs and
   # traces when it's not populated by instrumentation libraries.
-  # If enabled, make sure to enable this processor in a pipeline.
   # For more information, see https://docs.splunk.com/Observability/gdi/opentelemetry/components/resource-processor.html
-  #resource/add_environment:
-    #attributes:
-      #- action: insert
-        #value: staging/production/...
-        #key: deployment.environment
+  resource/add_environment:
+    attributes:
+      - action: insert
+        value: default
```

Comment:
For me it feels like it will pull a default value from somewhere, not that the value will be `default`. That said I don't have a good alternative, even a different value like `default_environment` doesn't improve much.

Reply:
"Default" might have a pre-existing meaning for customers, too.

Could we call it something else perhaps? Something that signals that the environment hasn't been defined by instrumentation libraries? 

Some ideas:

- `empty`
- `undefined` (Prolly not, as it has meanings in programming languages)
- `none`